tests/formatters/test_human_readable_formatter.py

```diff
@@ -29,10 +28,10 @@ def test_human_readable_formatter_model(
     stdout = capsys.readouterr().out
     assert (
         stdout
-        == """🥇 \x1B[1mmodel1\x1B[0m (score: 10.0)
-    \x1B[1;32mOK  \x1B[0m tests.conftest.rule_severity_low
-    \x1B[1;31mERR \x1B[0m tests.conftest.rule_severity_medium: Oh noes
-    \x1B[1;33mWARN\x1B[0m (critical) tests.conftest.rule_severity_critical: Error
+        == """🥇 \x1b[1mmodel1\x1b[0m (score: 10.0)
+    \x1b[1;32mOK  \x1b[0m tests.conftest.rule_severity_low
+    \x1b[1;31mERR \x1b[0m tests.conftest.rule_severity_medium: Oh noes
+    \x1b[1;33mWARN\x1b[0m (critical) tests.conftest.rule_severity_critical: Error
```

Comment:
Interesting, I know hex is not case sensitive, but curious how this changed 🤔

Reply:
Appears it's my local `ruff` kicking in action.

Even though ruff is configured for this project, it's on 0.2.2 and locally have 0.6.1 running.

Will revert this change.